commune_app/tests/test_chores.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+from commune_app import models
+from commune_app.models import User, Chore
+
+
+USER1_ID = 206609999
+USER1_USERNAME = "Elias"
+USER1_PASSWORD = "Wtf123"
+USER1_FIRST_NAME = "Elias"
+USER1_LAST_NAME = "Assy"
+USER1_EMAIL = "eliasakahaxor@gmail.com"
+
+CHORE_TITLE = "Dishes"
+CHORE_DATE = (1997, 10, 19)
+CHORE_BUDGET = 30
+CHORE_DESCRIPTION = "Clean the Dishes"
+CHORE_ASSIGN_TO = "Elias"
+CHORE_PASSED = True
+CHORE_STATUS = True
+
+
+@pytest.fixture
+def user0():
+    return User(
+        id=USER1_ID,
+        username=USER1_USERNAME,
+        password=USER1_PASSWORD,
+        first_name=USER1_FIRST_NAME,
+        last_name=USER1_LAST_NAME,
+        email=USER1_EMAIL
+    )
+
+
+@pytest.fixture
+def chore0(user0):
+    return models.Chore(
```

Comment:
Here you can do it like:
```suggestion
    return Chore(
```

Reply:
done.